spec/feature/mission_spec.rb

```diff
@@ -26,6 +26,41 @@ def fill_in_form(mission)
     end
   end
 
+  describe 'Sorting mission' do
+    let!(:first_mission) { create(:mission) }
+    let!(:second_mission) { create(:mission, created_at: DateTime.now + 1, ended_at: DateTime.now + 2) }
+    let!(:third_mission) { create(:mission, created_at: DateTime.now + 2, ended_at: DateTime.now + 3) }
+
+    context 'when goes to root' do
```

Comment:
可以的話請精準敘述 context ，現在只是 misisons#index 剛好是 project 的 home page ，等到後面步驟時 home page 可能會變動。
```suggestion
    context 'when visiting mission list' do
```